package.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "@speech/microsoft-speech-browser-sdk",
-  "version": "0.0.3",
+  "version": "0.0.1",
   "description": "Microsoft Speech SDK for browsers",
   "main": "distrib/speech.browser.sdk.js",
+  "types": "distrib/speech.browser.sdk.d.ts",
```

Comment:
types is required to get the npm package working with other typescript projects.